### DISCUSSION TITLE
Loading Issues

### DISCUSSION THREAD

**Discussion by cyn-hull** [iotaledger/firefly#2441] [Help & Support] [answered]
After installing the desktop app, I transferred a small test amount to the wallet. Since that transfer, the app will not load my account. I click the wallet, and the application just sits in loading mode, and never loads the wallet.  
How do I correct this issue?

**Comment by MiKeF05** [comment, marked as answer]
See discussion #2338

**Reply by cyn-hull** [reply]
THANK YOU!
